webpages/settings/alternative.css

```diff
@@ -0,0 +1,9 @@
+@import url("../styles/colors.css");
+@import url("../styles/components/tooltips.css");
+@import url("../styles/components/badges.css");
+@import url("../styles/components/categories.css");
```

Comment:
Those imports (except for the font) are useless because index.css is always there.